### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by KillerSH** [manan858/Detecting-Violence#2]
Do you have the dataset for violent/non-violent hockey or any other dataset for this topic?

**Comment by manan858** [comment]
Yes, I have dataset but I need to upload it on google drive first. You can run this code for UCF101 action recognition dataset.
